data_api/api/tests.py

```diff
@@ -22,29 +23,30 @@ def setUp(self):
         self.urns = ['tel:1234', 'twitter:5678', '876565']
         self.temba_group = FakeTemba(uuid='090IOU98', name='test_group', size=1)
         self.temba_contact = FakeTemba(uuid='97976768', name='test_contact', urns=self.urns,
-                                 groups=[self.temba_group.uuid], fields={'name': 'test_field'}, language='en',
+                                 groups=[_id_to_dict(self.temba_group.uuid)], fields={'name': 'test_field'}, language='en',
```

Comment:
Am not sure what changed but all these fields fail on validation in the test because they're expecting a `dict` but getting a `string`. I looked a bit at [some of the queries](https://github.com/rapidpro/data_api/blob/master/data_api/api/models.py#L93-L100) and it seemed like this was how they were expected now. Though is a bit of a guess as I haven't actually got data in my system apart from this test at the moment.

I suspect maybe the test is just out of date as there are also more obvious issues of strings/booleans not being typed correctly?

Reply:
looks like tests haven't been changed since the initial commit (https://github.com/rapidpro/data_api/commits/master/data_api/api/tests.py) so I'd guess this is an out-of-date test issue